shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/extensions/SharedDate+Extensions.kt

```diff
@@ -0,0 +1,15 @@
+package com.kcc.kmmhackathon.shared.utility.extensions
+
+import com.kcc.kmmhackathon.shared.utility.SharedDate
+
+fun SharedDate.isAfter(other: SharedDate): Boolean {
+    return this.compareTo(other) > 0
+}
+
+fun SharedDate.isBefore(other: SharedDate): Boolean {
+    return this.compareTo(other) < 0
+}
+
+fun SharedDate.isEqual(other: SharedDate): Boolean {
+    return this.compareTo(other) == 0
+}
```

Comment:
Added these because every time I looked at the `compareTo` code in the API file it made me go 🤯  and I was the one who wrote it